### PR TITLE
feat: check iceberg database, table and support desc in sparkcolumn

### LINKED ISSUE #291: Support orderby with is_asc parameter for OpenMLDB Batch
**Is your feature request related to a problem? Please describe.**

Currently OpenMLDB Batch only support `is_asc` as `true` for any OrderByNode which is for `WindowAggNode`. We should support reading this parameter and set `Spark Column` to use it. Refer to the code in `PhysicalNodeUtil.scala`.

```
  def getOrderbyColumns(windowAggNode: PhysicalWindowAggrerationNode, inputDf: DataFrame)
  : mutable.ArrayBuffer[Column] = {
    val windowOp = windowAggNode.window()

    val orders = windowOp.sort().orders()
    val ordersExprListNode = orders.getOrder_expressions_
    val orderByCols = mutable.ArrayBuffer[Column]()

    for (i <- 0 until ordersExprListNode.GetChildNum()) {
      val orderExpr = orders.GetOrderExpression(i)
      val colIdx = SparkColumnUtil.resolveOrderColumnIndex(orderExpr, windowAggNode.GetProducer(0))
      val column = SparkColumnUtil.getColumnFromIndex(inputDf, colIdx)
      // TODO: Support reading is_asc from physical node, currently use asc by default
      orderByCols += column.asc
      /*
      if (orders.is_asc()) {
        orderByCols += column.asc
      } else {
        orderByCols += column.desc
      }
      */
    }

    orderByCols
  }
```

**Describe the solution you'd like**

Get the parameter from physical plan and set the Spark Column object.

**Describe alternatives you've considered**

None.

**Additional context**

None.

### PR DESCRIPTION
**Pull request by zekai427**
- Create Iceberg database. If database not exists, will create a new database.
- Check table. If table already exists, will throw a ERRO
- Support desc in SparkCoulumn
- Resolve https://github.com/4paradigm/OpenMLDB/issues/291

### CODE REVIEW

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala [hunk 1]
```diff
@@ -280,6 +279,21 @@ class OpenmldbSession {
...
+    try {
+      catalog.createNamespace(namespace)
+    } catch {
+      case _: org.apache.iceberg.exceptions.AlreadyExistsException =>
+    }
+
+    // Check if table exists
+    if(catalog.tableExists(tableIdentifier)) {
+      catalog.close()
+      logger.error("Table %s already exists".format(tableName))
```
Comment (tobegit3hub) [inline java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala:296]: throw `HybridSeException` if table exists.

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala [hunk 2]
```diff
@@ -93,15 +93,11 @@ object PhysicalNodeUtil {
       val orderExpr = orders.GetOrderExpression(i)
       val colIdx = SparkColumnUtil.resolveOrderColumnIndex(orderExpr, windowAggNode.GetProducer(0))
       val column = SparkColumnUtil.getColumnFromIndex(inputDf, colIdx)
-      // TODO: Support reading is_asc from physical node, currently use asc by default
-      orderByCols += column.asc
-      /*
-      if (orders.is_asc()) {
+      if (orderExpr.is_asc()) {
```
Comment (tobegit3hub) [inline java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala:96]: Add the unit tests to verify the config works.

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala (hunk 1 above)
Comment (zekai427) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala:296]: solve

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala (hunk 2 above)
Comment (zekai427) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala:96]: Add the unit tests in next PR

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by dl239 — APPROVED** [review]

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala (hunk 2 above)
Comment (zekai427) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/PhysicalNodeUtil.scala:96]: Have solve in https://github.com/4paradigm/OpenMLDB/pull/296

**Pull request merged**